src/Workspaces/Core/Portable/Shared/Utilities/BloomFilter.cs

```diff
@@ -325,7 +326,7 @@ public void Add(string value)
 
     private int GetBitArrayIndex(string value, int i)
     {
-        var hash = ComputeHash(value, i);
+        var hash = ComputeHash(value, i, _isCaseSensitive);
         hash %= _bitArray.Length;
         return Math.Abs(hash);
```

Comment:
See how the result is modded by this array' length?  I don't think you can use these values across Bloom filters.  

I *do* think you could reuse data produced by a static computation (like ComputeHash)

Reply:
dang it, I moved ComputeHash to be static so that I wouldn't have dependencies like this. Let me move things around to see if I can address that.